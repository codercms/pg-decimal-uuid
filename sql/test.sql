CREATE EXTENSION uuid_decimal;
SELECT decimal_to_uuid(uuid_to_decimal('01923554-9e91-d520-f927-46af2232648c'));
SELECT decimal_to_uuid(uuid_to_decimal('00000000-0000-0000-0000-000000000000'));
SELECT decimal_to_uuid(uuid_to_decimal('ffffffff-ffff-ffff-ffff-ffffffffffff'));
