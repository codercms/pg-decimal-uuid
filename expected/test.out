CREATE EXTENSION uuid_decimal;
SELECT decimal_to_uuid(uuid_to_decimal('01923554-9e91-d520-f927-46af2232648c'));
           decimal_to_uuid            
--------------------------------------
 01923554-9e91-d520-f927-46af2232648c
(1 row)

